source/manual/pagerduty.html.md

```diff
@@ -1,13 +1,29 @@
 ---
 owner_slack: "#govuk-2ndline"
-title: PagerDuty drill
-parent: "/manual.html"
+title: PagerDuty
+section: 2nd line
+type: learn
 layout: manual_layout
-section: Icinga alerts
-last_reviewed_on: 2019-07-18
-review_in: 6 months
+parent: "/manual.html"
+last_reviewed_on: 2019-08-16
+review_in: 3 months
 ---
 
+Some alerts are urgent enough to warrant immediate attention, such as parts of the site becoming
+unavailable or large quantities of error pages being served. We use [PagerDuty](https://governmentdigitalservice.pagerduty.com)
+to escalate to these people (in order), mirroring the out-of-hours escalation order:
+
+1. Primary Engineer
+2. Secondary Engineer
+3. Designated Programme team member (might not be technical)
+
+When an alert that triggers PagerDuty goes off, someone on the escalation schedule must acknowledge
```

Comment:
Given the scope of the page has grown to PagerDuty in general, I'd perhaps move this paragraph and the one below to below the `## PagerDuty drill` heading, as they're specific to the drill.

Reply:
Have moved the paragraph below into that section (`This happens every Wednesday morning at 10am.`). But _this_ paragraph isn't specific to the drill, is it? It describes what happens when a PagerDuty alert happens (whether it's a drill or not).

Reply:
Ah, yeah, I'm getting confused by what the "escalation schedule" and the "Programme Team Escalations" schedule are. I read this as someone on the "Programme Team Escalations" schedule must acknowledge the incidents.